File: Python/Beginners/seq_logicaII.py
# Caio Cesar 02/12/23
# BEE 1145

valores = input().split()

valor_x = int(valores[0])
valor_y = int(valores[1])

x = 1

for y in range(1, valor_y + 1):
    if x == valor_x:
        print(y)
        x = 1
    else:
        print(y, end= " ")
        x += 1

File: C++/Beginner/seq_logicaII.cpp
// Caio Cesar 02/12/23
// BEE 1145

#include <iostream>
using namespace std;

int main(){
    int valor_x, valor_y, x = 1, y;

    cin >> valor_x >> valor_y;

    for(y = 1; y < valor_y + 1; y++){
        if(x == valor_x){
            cout << y << endl;
            x = 1;
        }else{
            cout << y << " ";
            x++;
        }
    }
}